testing/src/main/java/io/stargate/it/http/RestApiv1Test.java

```diff
@@ -705,6 +700,41 @@ public void addRow() throws IOException {
     getRow(tableName, rowIdentifier);
   }
 
+  @Test
+  public void addRowValidation() throws IOException {
+    String tableName = "tbl_addrow_" + System.currentTimeMillis();
+    createTable(tableName);
+
+    okhttp3.Response response =
+        RestUtils.postRaw(
+            authToken,
+            String.format("%s:8082/v1/keyspaces/%s/tables/%s/rows", host, keyspace, tableName),
+            "",
+            422);
+    Map<String, String> map = objectMapper.readValue(response.body().string(), Map.class);
+    assertThat(map.get("description")).isEqualTo("Request invalid: Row data cannot be null.");
+
+    response =
+        RestUtils.postRaw(
+            authToken,
+            String.format("%s:8082/v1/keyspaces/%s/tables/%s/rows", host, keyspace, tableName),
+            "{}",
+            422);
+    map = objectMapper.readValue(response.body().string(), Map.class);
+    assertThat(map.get("description"))
+        .isEqualTo("Request invalid: Columns cannot be null or empty.");
+
```

Comment:
one more case here for `  "{\"columns\":[]}"`

Reply:
After the fix this case works as below. I will add this case also to the integration test.
curl --location --request POST '127.0.0.2:8082/v1/keyspaces/myworld/tables/mytable/rows' --header "X-Cassandra-Token: $AUTH_TOKEN" \
--header 'Content-Type: application/json' \
--data '{
"columns":[]    }'
{"description":"Request invalid: Columns cannot be null or empty.","code":422}